tests/cli/test_ci.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+# Note: execute this file from the project root directory
+
+set -e
+
+. /usr/share/beakerlib/beakerlib.sh
+. $(dirname $0)/lib/lib.sh
+
+CLI="${CLI:-./src/bin/composer-cli}"
+
+dnf -y install \
```

Comment:
We need a better way to manage these test dependencies, probably a file similar to a pip requirements file. There should be a single place to edit the list.

Reply:
+1 for the single place for dependencies. 

However many of the test jobs don't need everything, which means they will still incur the penalty of network downloads and dnf install for things not needed in the particular job. If this is acceptable then we've already got this  place, which is `test/vm.install`.

Reply:
FTR I'm starting to like the fact that the test script makes sure it has all the packages it needs and this doesn't affect the rest of the test suite (i.e. not in vm.install and affecting all scenarios). I think this is acceptable for now so marking this comment as resolved.